<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>10원</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_50">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>50원</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_100">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>100원</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_500">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>500원</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCoffee">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee(100원)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTea">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea(150원)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMike">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Mike(200원)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
